configurations/ldap-authorization-and-authentication.yaml

```diff
@@ -0,0 +1,8 @@
+authenticator: 'com.scylladb.auth.SaslauthdAuthenticator'
+prepare_saslauthd: true
+use_saslauthd_authenticator: true
+authenticator_user: 'cassandra'
+authenticator_password: 'cassandra'
+
+authorizer: 'CassandraAuthorizer'
```

Comment:
We need to change it to the LDAP authoriser AFAIK.
right @fgelcer ?

Reply:
no, currently we still use `CassandraAutorizer` when we need to do authorization.
for LDAP authorization we need to set:
```
{
'role_manager': 'com.scylladb.auth.LDAPRoleManager',
'ldap_url_template': ldap://{ldap_server_ip}:{ldap_port}/LDAP_BASE_OBJECT}?cn?sub?(uniqueMember=uid={{USER}},ou=Person,{LDAP_BASE_OBJECT})', # this can be different, depending on how the user wants to configure the LDAP configuration DB
'ldap_attr_role': 'cn', # same as above
'ldap_bind_dn': f'cn=admin,{LDAP_BASE_OBJECT}', # same as above
'ldap_bind_passwd': LDAP_PASSWORD
}
```

Reply:
@dekimir can confirm, but we don't have any other "Authorizer" but `CassandraAuthorizer`

Reply:
> We need to change it to the LDAP authoriser AFAIK.

This is not true.  As the docs explain, you just need to change the authenticator.